docs/devguides/contribution_guide.rst

```diff
@@ -56,56 +74,27 @@ How to contribute:
 #. Please, **update the documentation**, if you committed significant changes
    to our code. 
 
-#. **Commit your changes and push** your feature branch to your GitHub fork.
-   Don't forget to reference the GitHub issue associated with your task.
-   Squash your commits into a single commit with git's interactive rebase.
-   Create a new branch if necessary.
+#. **Commit your changes and push** your feature branch to your GitHub fork:
 
     .. code:: bash
 
         git add my_files
         git commit -m "fix: resolve issue #271"
-        git push origin my_branch
+        git push origin what_my_code_does_branch
 
     Follow the `semantic commit notation <https://seesparkbox.com/foundry/semantic_commit_messages>`_
     for the name of the commit.
 
-#. **Create a new pull request** to get your feature branch merged into dev
-   for others to use. You’ll first need to ensure your feature branch contains
-   the latest changes from dev. 
-
-    .. code:: bash
-
-        # (external contribs): make a new pull request:
-
-        # merge latest dev changes into your feature branch
-        git fetch origin
-        git checkout dev
-        git pull origin dev 
-        git checkout my_branch
-        git merge dev  # you may need to manually resolve merge conflicts
-
-#. Once your change has been successfully merged, you can **remove the source
-   branch** and ensure your local copy is up to date:
-
-    .. code:: bash
-
-        git fetch origin
-        git checkout dev
-        git pull
-        git branch -d my_branch
-        git branch -d -r origin/my_branch
+#. Create a new `pull request <https://github.com/deepmipt/DeepPavlov/pulls>`_
+   to get your feature branch merged into dev for others to use.
+   Don't forget to reference the GitHub issue associated with your task in the description.
```

Comment:
probalby add: 

see the [references how-to](https://guides.github.com/features/issues/#notifications) for info on how to reference an issue in your PR.